### DISCUSSION TITLE
新聞：農業技術應用🌾

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#48] [General]
**新聞大綱:**
中興大學土木系特聘教授**楊明德團隊**從莫拉克風災開始，結合無人機與人工智慧技術，在台灣山林的坡地監測及防災調查。現今更進一步與農業專家合作，協助農民從事「**精準農業**」。
「精準農業」須透過「**卷積神經網路（CNN）**」模仿人類大腦對「顏色辨識認知」，利用無人機的圖像資料，判斷農作物生長狀況的差異性。
研究團隊發展技術概略如下:

1. 「植株定位技術」，結合AI物件偵測及影像處理技術，準確率高達九成。

2. 「陸空協作之水稻最佳收獲模式」，透過無人機回傳影像資料後，再以演算法推估穀粒含水量，整合天氣資料，準確提供最適合的收成排程及順序。

3. 「生長監測」，系統已經能精準的針對單株農作物進行株高變化、葉綠素含量、含氮量等大數據分析，讓農民可以決定何處、何時需要施肥。

目前正整合「邊緣計算」進行快速資料提供及即時判讀，按照SOP進行分析並提供正確判斷，不僅減少資料負荷量，也讓花費時間縮短。而Uber的操作模式也讓楊明德有新的構想，建置雲端資料處理平台，讓農民在監測農作物生長及災損，更精準迅速獲得正確資料，也計劃推動「**無人機Uber**」，不需要專業團隊到現場去操作，資料庫建置完成後，使用者及資料提供者都能從中獲取好處。

**心得:**
我覺得研究團隊的出發點很好，希望讓農民不必再擔心種植的收穫會不會因為一場災情就失去辛苦種植的作物，而所做成的成果不僅是農業方面的降低損失及成本，也有成功做出高準確技術及迅速監測系統，對於整體農業帶來的收穫真的很高，也期待該研究團隊往後的發展。

**新聞來源:**
https://www.1111.com.tw/news/jobns/141809

![image](https://user-images.githubusercontent.com/90883526/136688859-6c805fc8-0941-443c-9ad8-272a3f50d252.png)
![image](https://user-images.githubusercontent.com/90883526/136688860-4c092434-a5d8-41e0-8ba3-646c75522035.png)

**Comment by Jay-722** [comment]
感覺農業現在早已不是一堆老人在田裡彎腰了，變得好高科技。

**Comment by jim11261314** [comment]
這就是美國高科技農夫的進階版，
我認為工廠也會進步成一人工廠
剩下靠機器人幫忙

**Comment by LanningYang** [comment]
現在已經是智慧農業 如自動灑水系統
